contracts/build/OnRamp.compiled.json

```diff
@@ -0,0 +1 @@
+{"hash":"cf1fb1b2a725c0a599ba9e1274d227872517d570b8e520d803d2f241e4d58537","hashBase64":"zx+xsqclwKWZup4SdNInhyUX1XC45SDYA9LyQeTVhTc=","hex":"b5ee9c7241021c01000255000114ff00f4a413f4bcf2c80b0102016202170202cc030a0201200409020120050600b9420c700915be001d0d3030171b0915be0fa403001d72c200000001496d4fa4081008b8e11d72c200000001c92f23fe1d46d5981008ce201d181008b01ba9358f00ae031d0d72c200000000cf2bfd33fd33fd4d4d4fa40d4d15506f00b80201200708000f0c4071c17cb82120001b1510cc3c0094c071c16cfcb8fa60001fd19103759f971f529086382f971f536c0201200b120201200c0f0201200d0e002f3b51343e9034c00064be90249b4078b4cff53d0134c3f4600047321401b3c5891ba58d00dc33c2c0261c73c2c0140133c5b884b2cff33d0032c3f27b5520020120101100db16fc021b0cc0f435cb08000000033caff4cff4cff535353e9035345505e560c1fc0317c12082bebc2001f43e903e900c7e900c747e0a208c04000001800000007232cfc5f2cfc572cfc4f333331400f3c585331400f3c5b260063232c15400f3c5807e809c73c2dab3325c3ec02000610c3c02151c42e0c1fc0308a90415841195840101d04cfc0344158411840d8409bc026040265505a1d14554110180fc03a0020120131602012014150027104cfd03dbe844bcbd343e9034cff4803d013460002932140133c584b2cff2803d0032741004c8fd058060005b408c8cb3f17cb3f71cf0b1f15cb3f13cb3fcccccc01cf16ccc98208630000c8cb1612cbf771cf0b61ccc970fb00802012018190053bc8afc683ac6deda5cc6d0c2d2dcd8d2dcd65ce8dedc5cc6c6d2e05c9edca4c2dae1045a9897181718440201201a1b0009b8eb9f82a8000dba8acf82af90080a0a3149"}
```

Comment:
I think these files shouldn't be committed and instead pulled from chainlink-ton, like we do on Solana